advocacy_docs/partner_docs/SIBVisionsGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Vision X'
+---
+
+Implementing VisionX with EDB Postgres Advanced Server and EDB Postgres Extended requires the following components:
+
+- EDB Postgres Advanced Server
+- EDB Postgres Extended
```

Comment:
EDB products should be one line.

EDB Postgres Advanced Server or EDB Postgres Extended Server